.github/workflows/dco.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       - name: DCO Check
         uses: tim-actions/dco@v1.1.0
         with:
-          commits: ${{ steps.get-pr-commits.outputs.commits }}
\ No newline at end of file
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
```

Comment:
Should the newline be there at the end of file?

Reply:
I exactly added it